app/players/knight.py

```diff
@@ -0,0 +1,9 @@
+from app.players.player import Player
+
+
+class Knight(Player):
```

Comment:
what was the point of making an abstract class of player and then not expanding  it here? Why not simply create a Knight class instead of Player and write less code?